styleguide/source/assets/icons/svg/icon-facebook-hover.svg

```diff
@@ -0,0 +1 @@
+<svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44" width="44" height="44" fill="#02aaec"><path d="M22 0a22 22 0 1 0 22 22A22 22 0 0 0 22 0zm5.46 13.49h-2.4c-1.88 0-2.24.89-2.24 2.2v2.89h4.48l-.58 4.53h-3.9v11.63h-4.68V23.12h-3.91v-4.53h3.91v-3.34c0-3.88 2.37-6 5.82-6a32.11 32.11 0 0 1 3.49.18v4.05z"/></svg>
```

Comment:
Instead of adding another svg for the hover couldn't we just change the `fill` in the scss? Meaning we'd save half the space on extra svgs and then we'd just set the hover state for the link to include a fill. Especially if they're all the same color on hover this seems like less work. This would likely also be easier to update should UX decide that they want to change the hover color. (Updating all in one line instead of having to get more svgs).